remoteworkitem/tracker_repository_blackbox_test.go

```diff
@@ -1,12 +1,18 @@
 package remoteworkitem_test
 
 import (
+	"os"
 	"testing"
 
 	"github.com/almighty/almighty-core/application"
 	"github.com/almighty/almighty-core/gormsupport"
 	"github.com/almighty/almighty-core/gormsupport/cleaner"
+	"github.com/almighty/almighty-core/migration"
+	"github.com/almighty/almighty-core/models"
 	"github.com/almighty/almighty-core/remoteworkitem"
+	"github.com/almighty/almighty-core/resource"
+	"github.com/almighty/almighty-core/workitem"
```

Comment:
A chore: what do you think about adding a line in btw our packages and the external ones ?